src/server_manager/electron_app/release_scripts/bump_electron_version.js

```diff
@@ -1,39 +0,0 @@
-// Copyright 2018 The Outline Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-// Increments the "version" field in the Outline Manager Electron app's
-// config.json file.
-
-const fs = require('fs');
-const semver = require('semver');
```

Comment:
I don't think we use semver elsewhere. We should remove it from our dependencies.

Reply:
Good catch. Done.